java/vector/src/main/java/org/apache/arrow/vector/util/ByteFunctionHelpers.java

```diff
@@ -17,14 +17,13 @@
 
 package org.apache.arrow.vector.util;
 
-import org.apache.arrow.memory.BoundsChecking;
-
 import io.netty.buffer.ArrowBuf;
-import io.netty.util.internal.PlatformDependent;
 
 /**
  * Utility methods for memory comparison at a byte level.
+ * This class will be removed. Please use org.apache.arrow.memory.util.ByteFunctionHelpers instead.
```

Comment:
nit: use the @deprecated tag.

Reply:
Thanks for the kind reminder. Revised accordingly.